packages/terra-form-select/CHANGELOG.md

```diff
@@ -29,6 +29,9 @@ Unreleased
 
 6.2.0 - (April 22, 2020)
 ------------------
+### Added
+* Added `NativeSelect` implementation to replace the default select variant.
+
```

Comment:
Nit: This was bumped into a released section

Reply:
Updated.